coalib/bears/Bear.py

```diff
@@ -249,8 +249,14 @@ def execute(self, *args, **kwargs):
             result = self.run_bear_from_section(args, kwargs)
             return [] if result is None else list(result)
         except (Exception, SystemExit):
-            self.warn('Bear {} failed to run. Take a look at debug messages'
-                      ' (`-V`) for further information.'.format(name))
+            if self.kind():
```

Comment:
yes please definitely do `if self.kind() == BEAR_KIND.XYZ`, relying on those values is bad as we use an enum which auto-generates those values^^